src/Pages/Entry.cshtml.cs

```diff
@@ -14,17 +15,28 @@ namespace ePiggyWeb.Pages
     public class EntryModel : PageModel
     {
         private EntryDatabase EntryDatabase { get; }
-        public EntryModel(EntryDatabase entryDatabase)
+        private GoalDatabase GoalDatabase { get; }
+        private UserDatabase UserDatabase { get; }
+        private EmailSender EmailSender { get; }
+        private int UserId { get; set; }
+        public EntryModel(EntryDatabase entryDatabase, GoalDatabase goalDatabase,UserDatabase userDatabase, EmailSender emailSender)
         {
             EntryDatabase = entryDatabase;
+            GoalDatabase = goalDatabase;
+            UserDatabase = userDatabase;
+            EmailSender = emailSender;
+            UserDatabase.Deleted += OnDeleteUser;
+        }
+
+        private async void OnDeleteUser(object sender, UserModel user)
+        {
+            await EmailSender.SendFarewellEmailAsync(user.Email);
         }
 
```

Comment:
As I undertand we need delete user button somewhere?

Reply:
That would be nice to have, maybe on the header or footer for now, just to show that we have this sort of functionality